css/styles.css

```
@@ -0,0 +1,163 @@
+body {
+    background-color: #fefae0
+  }
+  .nav {
+    display:  grid;
+    grid-template-columns: 4fr 1fr 1fr;
+    margin-left: 6vw;
+    color: #606c38;
+    }  
+  
+  .headline-tag {
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: column;
+    align-items: center;
+    height: 32vh;
+    font-family: 'Roboto', sans-serif;
+    color: #606c38
+    }
+    
+    .headline-tag sub {
+      height: 5vh;
+      font-weight: bold;
+      text-transform: capitalize;
+    }
+  
+    .headline-tag a {
+      border: solid grey;
+      border-radius: .5em;
+      color: black;
+      display: inline-block;
+      padding: 4px 14px;
+      text-decoration: none;
+      line-height: 35px;
+      background-color: #dda15e
+    }
+  
+    .headline-tag a:hover {
+      background-color: #bc6c25;
+      font-weight: bold;
+    }
+  
+    .headline-tag:after, .sponsors:after, .quote2:after, .why:after, .lorem:after{
+      content: "";
+      display: block;
+      border: 1px solid grey;
+      width: 80%;
+      margin: 20px auto;
+    }
+  
+    .lorem, .quote2 {
+      font-style: italic; 
+      font-weight: bold;
+    }
+  
+    .lorem blockquote:before, .quote2 blockquote:before {
+      content: open-quote;
+    }
+    
+    .lorem blockquote, .quote2 blockquote {
+      margin-left: auto;
+      margin-right: auto;
+      width: 45%;    
+    }
+  
+    .lorem blockquote:after, .quote2 blockquote:after {
+      content: close-quote;
+    }
+  
+    .lorem p, .quote2 p {
+      width: 30%;
+      margin-left: auto;
+    }
+  
+    .lorem p:before, .quote2 p:before {
+      content: '--'
+    }
+  
+    .why h2 {
+      text-align: center;
+      font-family: 'Roboto', sans-serif;
+      color: #606c38
```

```suggestion
      color: #606c38
```

Remember to add semi-colons even if it is the last declaration in the rule-set.